Sources/UIViewAPI/UIViewAPI.swift

```diff
@@ -20,4 +20,24 @@ public extension UIView {
     }
 }
 
+public extension UILabel {
+    var getIntrinsicLabelWidth: CGFloat {
```

Comment:
Please add a documentation what it does and a example snippet :)

Reply:
Renamed and added description in documentation

Reply:
Please add id also in the code. That way "quick help" will have something to show :)

Reply:
Fix for quick help :)